HW1

agurianova opened the pull request:
This HW about work with git.

Lochness04 committed:
Three txt files have been added

agurianova committed:
Organized files into a new folder

agurianova committed:
Added changes to the hw1.txt

agurianova committed:
Added changes to the test_revert.txt

agurianova committed:
Merge branch 'testing' into HW1

agurianova committed:
Revert "Merge branch 'testing' into HW1"

This reverts commit a53054c4b9d7ce4aa2f8d4e704c92297876c765e, reversing
changes made to 6a663f39a9a958e8fff4c20381ad5521803e51aa.

 Revert merging.

agurianova committed:
Added forgotten changes to the test_revert_merge.txt

agurianova committed:
Merge branch 'testing' into HW1
Merge branch 'testing' into HW1 x2

agurianova committed:
Revert "Revert "Merge branch 'testing' into HW1""

This reverts commit 22f3334ca3970987b302d189d1424a1d7525a39a.

Here I reverted first revert.

agurianova committed:
Added report and changes in README

rinnifox commented:
Hi Anastasia! Good job!
Just remove the `testing` branch as required by your home task.

One more comment: I see in your HW1 branch some `.DS_Store` file. Such files that are not related to repository contents should not appear in your remotes. To fix that, you can either create a `.gitignore` file indicating files/extensions that will be not tracked by git, or you can explicitly indicate the files to add instead of using `git add .` 

I'd recommend remove this unnecessary file and to add `.gitignore` to your HW1 branch. Then I will approve your PR to make all these changes also appear in your main branch.

agurianova committed:
Added .DS_Store to .gitignore

agurianova committed:
Removed .DS_Store from current directory

agurianova commented:
Hi Karina! Thank you for your attention and comments <3
I solved the authentication problem and deleted the testing branch. Then I added .DS_Store to .gitignore and deleted it. Hope it looks better now)
@rinnifox
